### ISSUE TITLE
Folder/Tabs

### ISSUE THREAD

**Issue by Taijai** [xgi/houdoku#189]
### Acknowledgements

- [X] Check that there is not an existing ticket for this request, and that it is not related to an existing ticket.
- [X] Create one ticket per feature. Feel free to create multiple tickets.
- [X] If this is an extension-related request, open a ticket in the [extensions repo](https://github.com/xgi/houdoku-extensions/issues/new/choose) instead.
- [X] Write a concise title for this ticket in the field above.

### Feature details

Would it be possible to create a tab section where we could organize the manga that we are reading. I understand that there is a filter tab that give us certain categories such as "unread" and "read." But having the ability to create our own folders where we can organize our manga in a way where everything isn't as cluttered would be nice. I generally like to organize my manga in folders such as "reading" "interested" "completed" and so on.

**Comment by xgi** [comment]
Yeah this is planned for the future, I'm expecting to add customizable "Categories" or "Groups" that series can be added to and filtered by. I don't think it will happen soon, though.
